src/note-API/auth-api.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const signUpAPI = (user) => {
+    return axios.post("/api/auth/signup", user);
+};
+
+const loginAPI = async (user) => {
+    return axios.post("/api/auth/login", user);
```

Comment:
Putting this in a try-catch block will be better.

Reply:
Oh yes, forgot it.